microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,376 @@
+        String baseUrl = "http://www.bing.com";
+        
+        driver.get(baseUrl);
+        WebElement elem = driver.findElement(By.id("sb_form_q"));
+        elem.sendKeys("WebDriver", Keys.RETURN);
+
+        System.out.println("Done");
+        driver.close();
+    }
+}
+```
+
+### [JavaScript](#tab/javascript/)
+
+```javascript
+const {Builder, By, Key, until} = require('selenium-webdriver');
+const {Options, ServiceBuilder} = require('selenium-webdriver/ie');
+
+(async function IEDriverSample() {
+  let ieService = await new ServiceBuilder('C:/<path to IEDriver download>')
+  let ieOptions = await new Options()
+  await ieOptions.setEdgeChromium(true)
+  await ieOptions.setEdgePath('C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe')
+
+  let driver = await new Builder().
+                         forBrowser('ie').
+                         setIeOptions(ieOptions).
+                         setIeService(ieService).
+                         build();
+  try {
+    await driver.get('http://www.bing.com');
+    await driver.findElement(By.id('sb_form_q')).sendKeys('WebDriver', Key.RETURN);
+    await driver.wait(until.titleIs('WebDriver - Bing'), 1000);
+  } finally {
+    await driver.quit();
+  }
+})();
+```
+
+* * *
+
+The following sections explain the steps in this sample in more detail.
+
+
+<!-- ====================================================================== -->
+## Create an InternetExplorerDriverService
+
+### [C#](#tab/c-sharp/)
+
+To create an `InternetExplorerDriverService`, use the `CreateDefaultService` method.  Pass in the following parameters:
+
+*  `dir`, which is the directory where you downloaded IEDriver to.
+*  `driverexe`, which is the name of the IEDriver executable.
+
+```csharp
+//change the path accordingly
+var dir = @"C:\<path to IEDriver download>";
+var driverexe = "IEDriverServer.exe";
+
+var ieService = InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+```
+
+### [Python](#tab/python/)
+
+Define `ieService` by calling `Service` from `selenium.webdriver.ie.service` with the path to the IEDriver executable you downloaded.
+
+```python
+from selenium.webdriver.ie.service import Service
+
+ieService = Service('C:/<path to IEDriver download>')
+```
+
+### [Java](#tab/java/)
+
+For Java, you do not need to create an InternetExplorerDriverService. You do need to specify the path to the IEDriver executable you downloaded.
+
+```java
+System.setProperty("webdriver.ie.driver","C:/<path to IEDriver download>");
+```
+
+### [JavaScript](#tab/javascript/)
+
+Define `ieService` by calling `ServiceBuilder` and passing it the path to the IEDriver executable you downloaded.
+
+```javascript
+let ieService = await new ServiceBuilder('C:/<path to IEDriver download>')
+```
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Define InternetExplorerOptions with additional capabilities for Microsoft Edge
```

Comment:
Is "InternetExplorerOptions" correct for every language below?  Change to something like "ieOptions"?  Maybe the wording is ok, if applies to most languages.